### DISCUSSION TITLE
Schema List Clients

### DISCUSSION THREAD

**Discussion by studio121-develop** [fattureincloud/api#416] [Q&A] [answered]
Dallo schema client https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/entities/clients trovod elle voci molto interessanti che però non sono compilabitli da fattureincloud

Nello specifico:

```
name: string┃null - Client name
type: enum┃null - ClientType: Client type
first_name: string┃null - Client first name
last_name: string┃null - Client last name
```

Sapete dirmi qualcosa di più? Perchè potrebbero essere utilissimi per integrazione con Odoo

**Comment by emajo** [comment, marked as answer]
Ciao @studio121-develop cosa intendi con`non sono compatibili con fatture in cloud`?

**Reply by studio121-develop** [reply]
Perdonami, è stata una mia grossolana svista nella schermata anagrafica del cliente. Risolto, possiamo chiudere
